vite.config.ts

```diff
@@ -12,13 +12,5 @@ export default defineConfig({
   ],
   preview: {
     port: 5173
-  },
-  server: {
-    //https://github.com/vitejs/vite/discussions/3396
-    host: true,
-    port: 5173,
-    hmr: {
-      clientPort: 8000 // nginx reverse proxy port
```

Comment:
I'm pretty sure this nginx stuff is also something that was added in the recently "dev env" PR.

Reply:
Yes,  this code was added for the dev-env to work, but it's breaking the `npm start` command. You can test it in the develop branch if you run `npm start` and you go to the browser

So I asked Guillermo about this and he said he'll work on fixing it, but for the moment this was a workaround to be able to use `npm start`

Should I comment the code instead of removing it?

Reply:
If we are basically reverting this PR about adding a docker compose dev environment...

- #87

... @MellyGray can you please add this under "Special notes for your reviewer"?

@GPortas should we re-open the following issue?

- #65

Or perhaps create a new issue and put it on the schedule?

Reply:
I just asked about this during a frontend meeting and new commits have been added to bring back the docker compose file. I'll take a look!

Reply:
@GPortas can you please create an issue for nginx? ^^